Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM openshift/origin-release:golang-1.10
+COPY . /go/src/github.com/openshift/cluster-samples-operator
+RUN cd /go/src/github.com/openshift/cluster-samples-operator && \
+    go build ./cmd/cluster-samples-operator
+
+FROM centos:7
+RUN useradd cluster-samples-operator
+USER cluster-samples-operator
+COPY --from=0 /go/src/github.com/openshift/cluster-samples-operator /usr/bin
```

Comment:
we need to COPY the content from https://github.com/openshift/cluster-samples-operator/tree/master/tmp/build/assets/arch as well

Reply:
thanks, fixed